### ISSUE TITLE
USB Error 100 after successfully clone before reboot

### ISSUE THREAD

**Issue by omexlu** [stevenshiau/clonezilla#56]
Hello,

I get a USB error in the console (100, I don't remember the exact error message) when the cloning was successful and I choose the reboot option.

However, the cloning was successful but the reboot does not work and I have to hardreset the PC.

I have had this problem with the current alternative-stable of Ubuntu on several PCs.

Is this perhaps a bug or how to solve the problem?

I did not have this problem with any of the previous versions.

Thanks in advance.

**Comment by stevenshiau** [comment]
Is this issue you encountered?

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=986721

**Comment by omexlu** [comment]
Yes, I remember seeing this error after clonezilla was finished and I wanted to reboot the system.

But also something with USB Error 100 which may be due to this problem.

So it seems that there is currently a problem here and no solution yet, right?

I was not able to reboot the system and had to do a hard reset on the PC.

**Comment by stevenshiau** [comment]
"So it seems that there is currently a problem here and no solution yet, right?" -> Basically the answer is yes. Systemd can not handle the way that Debian live system has. I tried to fix live-boot but apparently I did not make it...

**Comment by omexlu** [comment]
Hm, then we have to live with it for now until this was fixed, hopefully this can neither harm the clonezilla clone, hard drives or USB 🙈

Then we just have to always hardreboot the PC or how do I get out of the loop after the completion of clonezilla?

**Comment by stevenshiau** [comment]
Normally if you wait long enough, it will eventually reboot/halt.
I suggest you choose the "To RAM" option in the boot menu. Then it should be less harmful when you do a hardreboot.

**Comment by omexlu** [comment]
I never sees this to ram option in boot menu, I think live USB always boot to ram 🙄

Do you think this error get fixed soon, because of a earlier version of clonezilla I never had this issue.

**Comment by stevenshiau** [comment]
To RAM is available both in the 1st and 2nd level of boot menu:

https://clonezilla.org/fine-print-live-doc.php?path=./clonezilla-live/doc/01_Save_disk_image/01-clonezilla-boot-menu.doc#01-clonezilla-boot-menu.doc

"Do you think this error get fixed soon, because of a earlier version of clonezilla I never had this issue." -> I do not think it will be fixed very soon unless the upstream has a good mechanism to deal with this issue.

Actually earlier version of Clonezilla live should have same issue, just that systemd exits faster.

**Comment by omexlu** [comment]
Ah okay, I always choose the first line so it's booting to ram to, right?

Maybe you can do somethink at clonezilla level? I I choose shutdown maybe this error don't occurs?

**Comment by stevenshiau** [comment]
"Ah okay, I always choose the first line so it's booting to ram to, right?" -> Yes, choose the menu which shows "To RAM" in the description.

"Maybe you can do somethink at clonezilla level? I I choose shutdown maybe this error don't occurs?" -> Well, so far there is not too much we can do about this issue. Just try to use the "To RAM" suggestion I mentioned.

**Comment by omexlu** [comment]
OK thank you for support, I will try the to ram the next time :)

**Comment by omexlu** [comment]
Other question:

Beyond the boot menu, Clonezilla will copy all files to the memory and will probably tell the user when the boot media can be removed (unexplained in the docs).

Is that right it tells me when I can remove the clonezilla USB and that way the problem mentioned don't occurs?

**Comment by stevenshiau** [comment]
Once it boots into the login prompt, or actually the TUI asks about language in Clonezilla live, you can remove the USB stick.

"that way the problem mentioned don't occurs?" -> Not really. The issue still remains, but it will reduce the chance that your file system is not unmounted correctly.
